tweakable_encryptor_test.go

```diff
@@ -6,6 +6,7 @@ package saltpack
 import (
 	"bytes"
 	"crypto/sha512"
+	"fmt"
```

Comment:
The code duplication between here and encryptor.go is getting out of hand, and I'm worried we may not be testing what we think we're testing. Ideas?